ci/scripts/r_windows_build.sh

```diff
@@ -97,15 +97,15 @@ if [ -d mingw32/lib/ ]; then
   mkdir -p $DST_DIR/lib/i386
   mv mingw32/lib/*.a $DST_DIR/${RWINLIB_LIB_DIR}/i386
   cp $MSYS_LIB_DIR/mingw32/lib/lib{thrift,snappy}.a $DST_DIR/${RWINLIB_LIB_DIR}/i386
-  cp $MSYS_LIB_DIR/mingw32/lib/lib{zstd,lz4,crypto,utf8proc,re2,aws*}.a $DST_DIR/lib/i386
+  cp $MSYS_LIB_DIR/mingw32/lib/lib{zstd,lz4,brotlicommon,brotlidec,brotlienc,crypto,utf8proc,re2,aws*}.a $DST_DIR/lib/i386
```

Comment:
Is there a reason for not using `brotli*` here and below?

Reply:
No reason not to. Fixed now.